helloWorld/settings.py

```diff
@@ -46,6 +46,7 @@
 ]
 
 MIDDLEWARE_CLASSES = [
+    'htmlmin.middleware.HtmlMinifyMiddleware',
```

Comment:
Why is it not required? I know html size is quite low for now but I included it for making it scalable.